astro-ts-mode.el

```diff
@@ -110,15 +110,13 @@
   (treesit-range-rules
    :embed 'tsx
    :host 'astro
-   :local t
```

Comment:
We can't drop these - it would basically be reverting e93161c, which would result in the same garbled parsing as before. I'll take a look at the behaviour in HEAD at some point and see if I can spot why the syntax highlighting is broken here.

Reply:
Okay. Emacs 31 won't be released soon, so it's fine for now.